go/vt/vtgate/vcursor_impl.go

```diff
@@ -694,6 +694,14 @@ func (vc *vcursorImpl) HasCreatedTempTable() {
 	vc.safeSession.GetOrCreateOptions().HasCreatedTempTables = true
 }
 
+func (vc *vcursorImpl) GetDBDDLPluginName() string {
+	return *dbDDLPlugin
+}
+
+func (vc *vcursorImpl) FindKeyspace(ks string) bool {
+	return vc.KeyspaceExists(ks)
```

Comment:
It looks like `KeyspaceExists()` reads from the `vschema.Keyspaces` map without a mutex, which implies that the map is never updated by another goroutine. Doesn't that mean the result of this call will never change, even if vtgate has refreshed vschema in the background?

Reply:
I changed that to see the executor view of VSchema. 
The e2e test confirms that you can do create/drop multiple times in order.

`RebuildVSchemaGraph` is necessary to update the executor's view.